engine/security/protect-socket.md

```diff
@@ -1,17 +1,41 @@
 ---
-description: How to setup and run Docker with HTTPS
-keywords: docker, docs, article, example, https, daemon, tls, ca,  certificate
+description: How to setup and run Docker with SSH or HTTPS
+keywords: docker, docs, article, example, ssh, https, daemon, tls, ca,  certificate
 redirect_from:
 - /engine/articles/https/
 - /articles/https/
+- /engine/https/
 title: Protect the Docker daemon socket
 ---
 
 By default, Docker runs through a non-networked UNIX socket. It can also
-optionally communicate using an HTTP socket.
+optionally communicate using SSH or a TLS (HTTPS) socket.
 
-If you need Docker to be reachable through the network in a safe manner, you can
-enable TLS by specifying the `tlsverify` flag and pointing Docker's
+## Use SSH to protect the Docker daemon socket
+
+The `docker` CLI can talk to a remote Docker daemon via SSH,
+by specifying `$DOCKER_HOST` to `ssh://<USERNAME>@<HOST>`.
+
+```console
+$ export DOCKER_HOST=ssh://root@example.com
+$ docker ps
+```
+
+The support for SSH is available since Docker Engine 18.09.
+
```

Comment:
I think we can remove this, as 18.09 is EOL, so any "current" version of docker now supports it (people shouldn't be using older versions as they're insecure now)

```suggestion
```

Reply:
updated